.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "vendor/php-date-formatter"]
 	path = vendor/php-date-formatter
-	url = git@github.com:kartik-v/php-date-formatter
+	url = https://github.com/kartik-v/php-date-formatter
```

Comment:
We sure about changeing from ssh to https ?

Reply:
No, it was something I was trying while working on the pup build. Switching it back unless it breaks the zip/build